packages/fungi/src/ContextChips/index.tsx

```diff
@@ -0,0 +1,46 @@
+import styled from "@emotion/styled";
+import { Chip, Variant as ChipVariant } from "@leafygreen-ui/chip";
+import Icon from "@evg-ui/lib/components/Icon";
+import { size } from "@evg-ui/lib/constants/tokens";
+import { ContextChip } from "../Context/context";
+
+export type ContextChipsProps = {
+  chips: ContextChip[];
+  dismissible: boolean;
+  onDismiss?: (chip: ContextChip) => void;
+};
+
+export const ContextChips: React.FC<ContextChipsProps> = ({
```

Comment:
Wait... should we use [rich links](https://github.com/mongodb/leafygreen-ui/blob/main/chat/rich-links/README.md) since they're chat-native? it seems like the href is optional (or could link back to the line number with an anchor?)

Reply:
OK i ended up getting this to work with scrolling on click but I had to do it in a different way to support it; the chips are now recorded in `ChatProvider` context rather than `message.metadata`

<img width="428" height="753" alt="Screenshot 2025-12-10 at 10 51 56 PM" src="https://github.com/user-attachments/assets/054dc3f1-3587-46e2-992d-17f768dc2231" />